### ISSUE TITLE
Question about masked car

### ISSUE THREAD

**Issue by sky-fly97** [georghess/voxel-mae#2]
Hello, it's a really great job. I have two question:
      1)  if you take most of the point cloud of a car in the scene, then can you predict the car?
      2)  The first task is to reconstruct a fixed number of point coordinates in every voxel, which is equivalent to getting the coordinates of so many points without any priors, and only by chamfer distance loss?

**Comment by georghess** [comment]
Hi,
Thanks for your interest!

1) I am not sure I understand the question. By "predict the car" do you mean predict its location or predict the points corresponding to the car? For the pre-training, we remove most of the point cloud and then ask the network to reconstruct the masked parts. For instance, this could be to reconstruct the point cloud of a car given only parts of the car. For the fine-tuning, we train the model on the entire point cloud and supervise it with regular 3DOD.

2) The prior for the points are their corresponding voxel centers, e.g., the network predicts the offset for the points within each voxel.
